sbnci/PlottingScripts/CompareDataDistributions.C

```diff
@@ -326,13 +326,11 @@ void CompareDataDistributions(TString gCurVersion="v07_06_00", TString gRefVersi
               hRef = (TH1D*)RefFile->Get(Form("%s/%s", s_branch_names.Data(), s_histo_names.Data()));
               hCur = (TH1D*)CurFile->Get(Form("%s/%s", s_branch_names.Data(), s_histo_names.Data()));
 
-              if (hRef->Integral() > 0 && hCur->Integral() > 0) {
+              if (useNormalised && hRef->Integral() > 0 && hCur->Integral() > 0)
                   hCur->Scale((hRef->Integral()+hRef->GetBinContent(0)+hRef->GetBinContent(hRef->GetNbinsX()+1))/(hCur->Integral()+hCur->GetBinContent(0)+hCur->GetBinContent(hCur->GetNbinsX()+1)));
-
-                  double maxext = getMax(hRef, hCur);
-                  hRef->SetMaximum(maxext);
-
-              }
+	      
+	      double maxext = getMax(hRef, hCur);
+	      hRef->SetMaximum(maxext);
           }
```

Comment:
I misses this in my previous review.
It looks like lines from 330 to 333 were the body of the if conditions at line 329.

Reply:
They used to be, my reason for changing this was that if the normalisation is turned off you still would like the the plotting to know about the tallest plot? It should almost always be the reference plot so wouldn't be a problem, however, its worth keeping for the odd scenario where it isn't.